src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -121,6 +121,9 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
     with self.argument_context('acr create') as c:
         c.argument('allow_metadata_search', arg_type=get_three_state_flag(), is_preview=True, help="Enable or disable the metadata-search feature for the registry. If not specified, this is set to disabled by default.")
 
+    with self.argument_context('acr create') as c:
+        c.argument('dnl_scope', options_list=['--dnlscope'], help='Domain name label scope will add a hash to the end of a login server name. The resulting login server name will be in the format <registry-name>-<hash>.azurecr-io. Default is Unsecure.', is_preview=True, arg_type=get_enum_type(AutoGeneratedDomainNameLabelScope))
```

Comment:
Can be changed to "will add a hash to the resource name on the login server name"?